### PR TITLE
CircleCI Publishing Job

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

**Comment by akoushke** [comment]
> image
> 
> these two commits are missing the trailing `)` on scope.

addressed

File: .circleci/config.yml [hunk 1]
```diff
@@ -58,6 +58,14 @@ jobs:
...
+    executor: main-executor
+    steps:
+      - checkout_from_workspace
+      - run:
+          name: Release
+          command: npm run release
+
 # execute the jobs in a orderly manner
 workflows:
   setup_test_release:
```
Comment (lalli-flores) [inline .circleci/config.yml]: Need to add the `release` job to the workflow?

File: package.json [hunk 2]
```diff
@@ -27,37 +27,52 @@
...
     "@webex/eslint-config-base": "^0.8.0",
     "babel-eslint": "^10.0.3",
+    "conventional-changelog-angular": "^5.0.3",
     "eslint": "^6.3.0",
     "eslint-config-prettier": "^6.2.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-mocha": "^6.1.0",
     "eslint-plugin-prettier": "^3.1.0",
-    "jest": "^24.9.0",
```
Comment (lalli-flores) [inline package.json]: I think you merged things incorrectly, we still need Jest and Jest configurations

File: package.json [hunk 3]
```diff
@@ -6,9 +6,9 @@
     "clear": "rm package-lock.json && rm -rf node_modules",
     "prettier": "prettier \"src/**/!(static)/**/*.*(js|jsx)\"",
     "prettier:write": "prettier \"src/**/!(static)/**/*.*(js|jsx)\" --write",
-    "test": "jest",
-    "test:watch": "jest --watch",
```
Comment (adamweeks) [inline package.json]: These two lines are used by the "test" job and probably why this build is failing on circle.

**Pull request merged**